### ISSUE TITLE
TimingProvider misses error property.

### ISSUE THREAD

**Issue by chrisguttandin** [webtiming/timingobject#27]
The spec mentions an `error` property of the `TimingProvider` on several occasions but the `TimingProvider` interface doesn't contain an `error` property.

http://webtiming.github.io/timingobject/#x5-1-create-a-new-timing-object
http://webtiming.github.io/timingobject/#x6-5-report-an-error

**Comment by ingararntzen** [comment]
You are quite right. Both error reporting and also readystate reporting is only indicated in the spec I think, with no real follow through, neither in the spec nor in the timingsrc implementation.
